### PR TITLE
Copy wspb into sdk in preparation for moving it out of main

### PR DESCRIPTION
**Pull request by jefferai**

### CODE REVIEW

**Review by tmessi — APPROVED** [review]

File: sdk/wspb/wspb.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+// messages via a websocket.
+package wspb
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"sync"
+
+	"github.com/hashicorp/boundary/internal/errors"
```
Comment (tmessi) [inline sdk/wspb/wspb.go:14]: Probably need to switch to just using the stdlib `errors` package instead.

Comment (jefferai) [reply sdk/wspb/wspb.go:14]: yes, thanks

**Comment by jefferai** [comment]
I can confirm that the failing test is a bit flaky locally as well so ignoring...

**Comment by jefferai** [comment]
At least, ignoring in this PR :-)

**Pull request merged**